Public/components/buttons/button.css

```diff
@@ -0,0 +1,61 @@
+@import url("/Public/components/alert/alert.css");
+.button-output{
+    border: 1px solid var(--primary-color);
+    display: flex;
+    padding: 24px;
+}
+.btn{
+    padding: 4px 16px;
+    margin-left: 8px;
+    border-radius: 4px;
+    cursor: pointer;
+}
+.btn-icon{
+    border:none;
+    font-size: 24px;
+    border-radius: 50%;
+    background-color: inherit;
+    padding: 5px;
```

Comment:
The button icons on hover are getting oval background, you can make it circular by giving appropriate top-bottom and left-right margin.